modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServiceDeployerApi.java

```diff
@@ -308,16 +315,19 @@ public Response purge(@PathVariable("id") String id) {
     @ResponseStatus(HttpStatus.OK)
     @Secured({ROLE_ADMIN, ROLE_CSP, ROLE_ISV, ROLE_USER})
     @AuditApiRequest(methodName = "getCspFromRequestUri")
-    public List<String> getAvailabilityZones(
+    public ResponseEntity<List<String>> getAvailabilityZones(
```

Comment:
can you please confirm if this change does not impact the response model on swagger UI?

Reply:
@swaroopar Hi,I have tested and currently, it will not have any impact on the front end